torch/onnx/__init__.py

```diff
@@ -27,9 +27,109 @@ def _export(*args, **kwargs):
     return result
 
 
-def export(*args, **kwargs):
+def export(model, args, f, export_params=True, verbose=False, training=False,
```

Comment:
please don't duplicate the doc. If we move the doc here, please delete the doc in utils.py.

Reply:
Delete the duplicate doc in onnx.utils.py